src/undomanager.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'jquery',
+    'underscore',
+    'tpl!vellum/templates/undo_alert',
+], function(
+    $,
+    _,
+    UNDO_ALERT
+) {
+    function alertShown() {
+        return $('.fd-undo-delete').length;
+    }
+
+    function createAlert() {
+        $('.fd-scrollable-tree').prepend(UNDO_ALERT);
```

Comment:
I think the location of this alert might be problematic in large forms where the user has scrolled the question tree.